java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -54,11 +55,12 @@
 @Beta
 public class SeleniumManager {
 
-    private static final Logger LOG = Logger.getLogger(SeleniumManager.class.getName());
+    private static final Logger log = Logger.getLogger(SeleniumManager.class.getName());
```

Comment:
We have a lot more `LOG` than `log`, so I think we do want to use `LOG` here, I'll change it back